base/asyncmap.jl

```diff
@@ -0,0 +1,187 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+
+"""
+    AsyncCollector(f, results, c...; ntasks=100) -> iterator
+
+Apply f to each element of c using at most 100 asynchronous tasks.
+For multiple collection arguments, apply f elementwise.
+Output is collected into "results".
+
+Note: next(::AsyncCollector, state) -> (nothing, state)
+
+Note: `for task in AsyncCollector(f, results, c...)` end is equivalent to
```

Comment:
backtick should include the `end` - also `next(::AsyncCollector, state)` above should be backtick-highlighted